bc3_importer/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "BC3 Connector",
+    "summary": """
+        BC3 Connector.""",
+    "author": "Binhex, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/vertical-construction",
+    "category": "Sales/Sales",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "depends": ["base", "sale", "project", "uom", "sale_management"],
```

Comment:
No need to put implicit dependencies:

```suggestion
    "depends": ["project", "sale_management"],
```